benchmarks/user/README.md

```diff
@@ -0,0 +1,233 @@
+# Benchmarking a Devito Operator
+
+## Running benchmarks
+
+`benchmark.py` implements a minimalist framework to evaluate the performance of
+a Devito Operator while varying
+
+* the problem size (e.g., shape of the computational grid)
+* the discretization (e.g., space- and time-order of the input/output fields),
```

Comment:
so and to of wavefields

Reply:
I think I prefer "input/output fields" because one day we may want to use this for something that is not waves